tools/sdk-generation-pipeline/Dockerfile

```diff
@@ -40,9 +40,9 @@ RUN rm packages-microsoft-prod.deb
 RUN wget https://dotnet.microsoft.com/download/dotnet/scripts/v1/dotnet-install.sh && chmod 777 ./dotnet-install.sh
 RUN bash ./dotnet-install.sh
 RUN bash ./dotnet-install.sh -c 3.1
-ENV DOTNET_ROOT=/root/.dotnet PATH=$PATH:/root/.dotnet
```

Comment:
Should we be updating to a newer dotnet Docker image instead of installing it ourselves?

Reply:
Actually never mind of this Docker needs to contain the tools for other all languages currently so it cannot start with one language Docker.